### DISCUSSION TITLE
API for dynamically adding and removing childrenspec to the supervisor

### DISCUSSION THREAD

**Discussion by leonlee2013** [ergo-services/ergo#142] [Ideas]
Similar to these Erlang APIs：[#start_child-2](https://www.erlang.org/doc/man/supervisor.html#start_child-2),[#terminate_child-2](https://www.erlang.org/doc/man/supervisor.html#terminate_child-2),[#delete_child-2](https://www.erlang.org/doc/man/supervisor.html#delete_child-2)
Here is an attempt to write some code to illustrate some of my idea：https://github.com/leonlee2013/ergo/commit/47f49ae1a6468814d5b4242899f955b92174f0c8

**Comment by halturin** [comment]
This feature will be introduced in the next release 3.0. Working on it now.
